src/client/src/orangehrmAdminPlugin/pages/languagePackage/languagePackage.vue

```diff
@@ -192,6 +193,10 @@ export default {
     onClickTranslate(item) {
       navigate('/admin/languageCustomization/{id}', {id: item.id});
     },
+    onClickExport(item) {
+      const url = `${window.appGlobal.baseUrl}/admin/viewLanguagePackage/languageId/${item.id}`;
```

Comment:
*

Reply:
this is okay. we have same usage everywhere else with downloads.